docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -24,8 +24,10 @@ Also, the data you provide to the Observability AI assistant is _not_ anonymized
 You need following to use the Observability AI Assistant:
 
 * {stack} version 8.9 and later.
-
-* An account with a third-party generative AI provider, which the Observability AI Assistant uses to generate responses. The Observability AI Assistant supports the `gpt-3.5-turbo` or `gpt-4` models of OpenAI and Azure OpenAI Service.
+* This feature requires an https://www.elastic.co/pricing[Enterprise subscription]
+* An account with a third-party generative AI provider that supports function calling. The Observability AI Assistant supports the following providers:
+** OpenAI `gpt-3.5`+  
```

Comment:
Scanning through this I thought for a sec we only support 3.5, perhaps we can clarify by explicitly mentioning gpt-4 as well? (Especially because we don't really want to recommend 3.5).

Reply:
I think we should also change  gpt-3.5 to gpt-3.5-turbo

Reply:
I think you may be looking at an outdated version?

right now it says 

```
* An account with a third-party generative AI provider that supports function calling. The Observability AI Assistant supports the following providers:
** OpenAI `gpt-4`+.
** Azure OpenAI Service `gpt-4`(0613) or `gpt-4-32k`(0613) with API version `2023-07-01-preview` or more recent. 
```

Reply:
woops, yes 🤓